src/core/Akka/Dispatch/ActorTaskScheduler.cs

```diff
@@ -109,7 +110,7 @@ public static void RunTask(Func<Task> asyncAction)
                                   }
                                   else
                                   {
-                                      context.Self.Tell(new ActorTaskSchedulerMessage(exception), ActorRefs.NoSender);
+                                      context.Self.Tell(new ActorTaskSchedulerMessage(exception,currentMessage), ActorRefs.NoSender);
```

Comment:
If the async operation failed, pass back the exception together with the current user message (captured above)